app/models/github_user.rb

```diff
@@ -0,0 +1,48 @@
+class GithubUser
+
+  attr_reader :attrs, :token
+
+  def initialize(attrs={}, token)
+    @attrs = attrs
+    @token = token
+  end
+
+  def nickname
+    @attrs[:login]
+  end
+
+  def self.search_user(token)
+    user = GithubService.new(token).search_user
+    new(user, token)
+  end
+
+  def avatar_url
+    attrs[:avatar_url]
+  end
+
+  def starred(token)
```

Comment:
@caward12 I think we can remove the parameter here since we set it up in initialize.